src/Widgets/calendar/GridDay.vala

```diff
@@ -56,20 +57,65 @@ public class DateTime.Widgets.GridDay : Gtk.EventBox {
 
         label.name = "date";
 
-        add (label);
+        main_grid = new Gtk.Grid ();
+        main_grid.halign = Gtk.Align.CENTER;
+        main_grid.orientation = Gtk.Orientation.VERTICAL;
```

Comment:
Adding an orientation here doesn't end up doing anything since you're packing widgets using `attach ()` instead of `add ()`

Reply:
Done.